security/client-certificate-validator/src/main/webapp/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
+<html>
+    <head>
+        <title>TODO supply a title</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <div>TODO write content</div>
```

Comment:
do you need to write some content?

Reply:
It's good tha tyou spotted this. I don't even know how this client-certificate-validator project got into this PR, I'll remove it.